src/game.py

```diff
@@ -0,0 +1,50 @@
+import sys
+from .players import AiPlayer, HumanPlayer
+from .board import Board
+
+class Game:
+
+    def __init__(self):
+        self.board = Board()
+        self.player1 = self.create_human_player()
+        self.player2 = self.create_ai_player()
+
+    def create_human_player(self):
```

Comment:
Why `Game` has to define how players are created? It should just prepare players like `self.player1 = Human(...)`.